Assets/Scripts/Player/PlayerSizeManager.cs

```diff
@@ -9,35 +9,57 @@ namespace Player
     /// </summary>
     public class PlayerSizeManager : MonoBehaviour
     {
+        private float playerScore=100;
+
+        private void UpdatePlayerScale()
+        {
+            //divide by 100 to get decimal percentile
+            var scaleIncrease = playerScore / 100;
```

Comment:
I understand why you're dividing by 100 here, but might be worth moving it to its own variable.

Reply:
it's not a magic number so it should be fine i think, unless you're really adamant

Reply:
Should be fine then, it's just a bit questionable due to how you are handling the scaling.